scripts/build_ngx_pagespeed.sh

```diff
@@ -44,10 +70,34 @@ Options:
       Print this message and exit."
 }
 
-# Prints an error message and exits with an error code.
-function fail() {
+RED=31
+GREEN=32
+YELLOW=33
+function begin_color() {
+  color="$1"
+  echo -e -n "\e[${color}m"
```

Comment:
Are you sure it's not portable?  Setting colors this way is standardized ([pdf](http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-048.pdf), p75) and is supported everywhere I'm aware of.